### PR TITLE
Fix reload mode

### PR DESCRIPTION
**Pull request by freddyaboulton**
## 🎯 PRs Should Target Issues

Before your create a PR, please check to see if there is [an existing issue](https://github.com/gradio-app/gradio/issues) for this change. If not, please create an issue before you create this PR, unless the fix is very small. 

Not adhering to this guideline will result in the PR being closed. 

## Tests

1. PRs will only be merged if tests pass on CI. To run the tests locally, please set up [your Gradio environment locally](https://github.com/gradio-app/gradio/blob/main/CONTRIBUTING.md) and run the tests: `bash scripts/run_all_tests.sh`

2. You may need to run the linters: `bash scripts/format_backend.sh` and `bash scripts/format_frontend.sh`

### CODE REVIEW

File: js/core/src/Blocks.svelte [hunk 1]
```diff
@@ -720,19 +720,19 @@
...
-			{target}
-			{theme_mode}
-			on:mount={handle_mount}
-			{version}
-			{autoscroll}
-			{max_file_size}
-			client={app}
-		/>
-		<!-- {/if} -->
+		{#if $_layout && app.config}
```
Comment (freddyaboulton) [inline js/core/src/Blocks.svelte:723]: This is the fix @pngwn but not sure why it was removed from #9339 so can you review?

Comment (pngwn) [reply js/core/src/Blocks.svelte:723]: It breaks SSR iirc.

Comment (freddyaboulton) [reply js/core/src/Blocks.svelte:722]: @pngwn It doesn't break SSR mode from what I checked. And I also am pushing up some tweaks to make reload mode work on SSR.

### SSR Mode
ssr_reload_mode



### NO SSR MODE
no_ssr_reload_mode

**Review by pngwn — APPROVED** [review]
Tested and works well. Thanks @freddyaboulton!

**Comment by freddyaboulton** [comment]
Thank you @pngwn !!

**Pull request merged**